File: LifeCycleSandBox/app/src/main/java/com/example/lifecyclesandbox/movie_list/Movie.java
package com.example.lifecyclesandbox.movie_list;

public class Movie {
    private String movieTitle;

    public Movie(String movieTitle) {
        this.movieTitle = movieTitle;
    }

    public String getMovieTitle() {
        return movieTitle;
    }

    public void setMovieTitle(String movieTitle) {
        this.movieTitle = movieTitle;
    }
}

File: LifeCycleSandBox/app/src/main/java/com/example/lifecyclesandbox/movie_list/Movie.kt
package com.example.lifecyclesandbox.movie_list

import com.google.gson.annotations.SerializedName

data class Movie(
        @SerializedName("id")
        val id: Long,
        @SerializedName("title")
        val title: String,
        @SerializedName("release_date")
        val releaseDate: String,
        @SerializedName("overview")
        val overview: String,
        @SerializedName("vote_average")
        val voteAverage: Double
)